packages/common/src/utils/challenges.ts

```diff
@@ -0,0 +1,147 @@
+import { ChallengeRewardID, OptimisticUserChallenge } from '../models'
+
+import { formatNumberCommas } from './formatUtil'
+
+export type ChallengeRewardsInfo = {
+  id: ChallengeRewardID
+  title: string
+  description: (amount: OptimisticUserChallenge | undefined) => string
+  fullDescription?: (amount: OptimisticUserChallenge | undefined) => string
+  progressLabel?: string
+  remainingLabel?: string
+  panelButtonText: string
+  verifiedChallenge?: boolean
+}
+
+export const challengeRewardsConfig: Record<
+  ChallengeRewardID,
+  ChallengeRewardsInfo
+> = {
+  referrals: {
+    id: 'referrals',
+    title: 'Invite your Friends!',
+    description: (challenge) =>
+      `Earn ${challenge?.amount} $AUDIO, for you and your friend`,
+    fullDescription: (challenge) =>
+      `Invite your Friends! You’ll earn ${challenge?.amount} $AUDIO for each friend who joins with your link (and they’ll get an $AUDIO too)`,
+    progressLabel: '%0/%1 Invites Accepted',
+    remainingLabel: '%0/%1 Invites Remain',
+    panelButtonText: 'Invite your Friends'
+  },
+  'ref-v': {
+    id: 'ref-v',
+    title: 'Invite your Fans',
+    description: (challenge) =>
+      `Earn up to ${formatNumberCommas(challenge?.totalAmount ?? '')} $AUDIO`,
+    fullDescription: (challenge) =>
+      `Invite your fans! You’ll earn ${challenge?.amount} $AUDIO for each fan who joins with your link (and they’ll get an $AUDIO too)`,
+    progressLabel: '%0/%1 Invites Accepted',
+    remainingLabel: '%0/%1 Invites Remain',
+    panelButtonText: 'Invite your Fans',
+    verifiedChallenge: true
```

Comment:
nit: booleans should follow `isXXX` or `hasXXX` naming convention per our styleguide

Reply:
true nice